backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -105,26 +105,10 @@ Spree.ready(function(){
 
   $('body').on('click', 'a.spree_remove_fields', function() {
```

Comment:
Could this be called on persisted items as well?

Reply:
No. "spree_remove_fields" is for nonpersistent rows, "delete-resource" for persistent.